src/client/utils/download/download.ts

```diff
@@ -36,7 +36,7 @@ export function getMIMEType(fileType: FileFormat) {
 function saveFile(part: string | Buffer, fileName: string, fileFormat: FileFormat, fileEncoding: string) {
   const type = `${getMIMEType(fileFormat)};charset=${fileEncoding}`;
   const blob = new Blob([part], { type });
-  fileSaver.saveAs(blob, `${fileName}.${fileFormat}`, true); // true == disable auto BOM
+  fileSaver.saveAs(blob, `${fileName}.${fileFormat}`, true);
```

Comment:
removed comment was quite useful, IMHO
keep comment or introduce named variable / named method parameter

Reply:
Comment wasn't useful. It didn't provide any additional knowledge. If you cmd-b into `saveAs` you will see that last parameter is named `disableAutoBOM`. You can also turn on "parameter hints" in idea.

Reply:
How to enable "parameter hints" on Github during review?